src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnector.java

```diff
@@ -170,6 +170,16 @@ public List<Map<String, String>> taskConfigs(int maxTasks) {
 
         LOGGER.debug("Configuring {} YugabyteDB connector task(s)", taskConfigs.size());
         closeYBClient();
+
```

Comment:
Is this better in a `finally` block? One way to avoid a massive whitespace change is to turn this function into a `Impl` function, create the connection, call `Impl` and in `finally` close the connection